domain/src/test/groovy/life/qbic/domain/usermanagement/policies/PasswordPolicySpec.groovy

```diff
@@ -16,7 +16,7 @@ class PasswordPolicySpec extends Specification {
 
         then:
         report.status() == PolicyStatus.FAILED
-        report.reason().equalsIgnoreCase("Password shorter than 8 characters.")
+        report.reason().equalsIgnoreCase("EncryptedPassword shorter than 8 characters.")
```

Comment:
Please double check that the `EncryptedPassword` change is undone. Replacements of `Password` by `EncryptedPassword` are not always correct here.

Reply:
Done 👍